Sources/Extensions/UIViewController+Child.swift

```diff
@@ -8,37 +8,57 @@
 import UIKit
 
 extension UIViewController {
+    /// Add a given controller to the current controller as a child view controller. The given
+    /// controller's view is added as a subview to the given `UIView`.
+    ///
+    /// - parameter childController: The controller to add to the current controller as a child.
+    /// - parameter to: The container view in which to add the child controller's view.
+    /// - parameter insets: The insets that give a some padding inside the container if needed.
+    public func add(childController controller: UIViewController, to view: UIView, insets: UIEdgeInsets = .zero) {
+        // Notify Child View Controller
+        controller.willMove(toParent: self)
+        // Add Child View as Subview
+        view.addSubview(controller.view)
+        // Add Child View Controller
+        addChild(controller)
+        // Configure Child View inside the container view.
+        controller.view.constraint(to: view, insets: insets)
+        // Notify Child View Controller
+        controller.didMove(toParent: self)
+    }
     
     /// Add a given controller to the current controller as a child view controller. The given
     /// controller's view is added as a subview to the given `UIView`.
-    public func add(childController controller: UIViewController, to view: UIView) {
+    ///
+    /// - parameter childController: The controller to add to the current controller as a child.
+    /// - parameter to: The container view in which to add the child controller's view.
+    /// - parameter safeAreaInsets: The insets that give a some padding inside the container's safe area.
+    @available(iOS 11.0, *)
+    public func add(childController controller: UIViewController,
```

Comment:
Volgens https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html is de volgorde eerst `addChild()`, dan pas `addSubview()`. Ze roepen ook niet expliciet `willMove(toParent:)` aan.

Weet niet wat de side effects zouden kunnen zijn? Mss eens checken?

Reply:
Ah idd, wist ik niet. De `didMove` wel precies.